lib/surface/components/live_file_input.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Surface.Components.LiveFileInput do
+  @moduledoc """
+  Wrapper around Phoenix LiveView's built-in
+  `live_file_input/2` function.
+
+  The parent LiveView, or LiveComponent must `allow_uploads` during mount:
+
+  ```elixir
+    def mount(socket) do
+      socket = allow_upload(socket, :test, accept: ~w(.json), max_entries: 1)
```

Comment:
```suggestion
      socket = allow_upload(socket, :avatar, accept: ~w(.json), max_entries: 1)
```

Just to be consistent with the doc ;)